frame/im-online/src/tests.rs

```diff
@@ -78,9 +85,9 @@ fn should_report_offline_validators() {
 				session_index: 2,
 				validator_set_count: 3,
 				offenders: vec![
-					(1, 1),
```

Comment:
and all of these can be reverted, if you keep using `type AccountId = u64` and modify your impl of `IdentifyAccount`.